fiftyone/core/odm/utils.py

```diff
@@ -657,3 +657,28 @@ class DocumentRegistryError(Exception):
 
 
 _document_registry = DocumentRegistry()
+
+
+def load_dataset(_id=None, name=None):
```

Comment:
Can you please use `id` instead of `_id` as the parameter name here? For consistency with how other methods name kwargs that refer to IDs. And since the implementation of this function allows the provided value to be either an `Objectid` or a `str` because you cast to `ObjectId` internally (which is 👍).

For context, *within* methods, I often use `_id` when the value is an `ObjectId` and `id` when the value is a `str`.

As for the "public" interface of functions, kwargs that start with `_` are undocumented parameters that are intended to convey to other developers that hey, if you're using this parameter, you're doing something slightly subtle/unintended, there's probably a better way.

Reply:
Also can you please fill out the docstring for this method with `Args` and `Returns` like the other methods in this module?

By convention, every non-private function in `fiftyone.core` should have a full docstring, even if the function is only intended to be called by developers, not end users.